.github/workflows/release-odh-dashboard.yml

```diff
@@ -134,8 +134,9 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: |
-          gh release create "${{ github.event.inputs.RELEASE_VERSION }}" \
+          gh release create "${{ github.event.inputs.RELEASE_VERSION }}-odh" \
               --repo="$GITHUB_REPOSITORY" \
-              --title="${{ github.event.inputs.RELEASE_VERSION }}" \
+              --title="${{ github.event.inputs.RELEASE_VERSION }}-odh" \
+              --target="${{ github.event.inputs.RELEASE_VERSION }}-odh-release" \
```

Comment:
what do we do about the previous tag? Do we leave it as it is coz it can be changed later? I think the previous tag is "auto" by default.

Reply:
Yeah I think this can be changed later after publishing the release notes